doc/en/developer/source/policies/community-process.rst

```diff
@@ -12,116 +12,58 @@ The GeoServer community process adheres to the following guidelines to ensure
 that it retains open development practices:
 
 * **Transparency** 
-  Project decisions are made in an open and transparent way. When a decision is
-  made it is clear as to why it was made.
+  Project decisions are made in an open and transparent way. Key decisions are presented *GeoServer Improvement Proposals* (:ref:`GSIP <gsip>`), discussed in public. When a decision is made it is clear as to why it was made.
```

Comment:
We are currently set up with "Any GitHub user can create and edit pages to use for documentation, examples, support, or anything you wish."

With this in mind I will add your clarification that proposals can be made by anyone.